tests/integration/targets/mso_schema_template_deploy_status/tasks/main.yml

```diff
@@ -584,21 +573,6 @@
       - status2_site_temp2.current.bds == []
       - status2_site_temp2.current.vrfs == []
 
-  - name: Check deployment status by querying site and non associated Template 1
-    cisco.mso.mso_schema_template_deploy_status:
-      <<: *mso_info
-      schema: '{{ mso_schema | default("ansible_test") }}'
-      template: Template1
-      site: 'aws_{{ mso_site | default("ansible_test") }}'
-      state: query
-    ignore_errors: true
-    register: status_site_temp3
-
-  - name: Verify status after querying site with non associated Template 1
-    ansible.builtin.assert:
-      that:
-      - status_site_temp3.msg == "Provided Template 'Template1' not associated with Site 'aws_ansible_test'."
-
```

Comment:
Should we move this test to a non-cloud site?

Reply:
Reverted the change and now it is part of `non-cloud` site task.